book/modules/module6.tex

```diff
@@ -115,7 +115,7 @@
 
 	$\mathcal W$ is \emph{not} a subspace. To see this, notice that $\vec v=\mat{1\\1}\in \mathcal W$,
 	but $0\vec v=\vec 0\notin\mathcal W$. Therefore, $\mathcal W$ is not closed under scalar multiplication
-	and so cannot be a subspace.
+	and so it cannot be a subspace.
```

Comment:
I don't think `it` is needed.

Reply:
I think putting a subject pronoun before the verb makes the subject clear. The sentence is less confusing this way. If you strongly prefer the original I can revert.